roles/ovirt-cluster-upgrade/tasks/main.yml

```diff
@@ -101,22 +93,22 @@
           scheduling_policy: "{{ cluster_scheduling_policy }}"
         when: cluster_policy.changed | default(false)
   
-      - name: Start again stopped VMs
+      - name: Start again pinned VMs
         ovirt_vms:
           auth: "{{ ovirt_auth }}"
           name: "{{ item }}"
+          state: running
         ignore_errors: yes
         with_items:
-          - "{{ stopped_vms | default([]) }}"
-  
-      - name: Start again pin to host VMs
+          - "{{ pinned_vms_down }}"
```

Comment:
Are we starting pinned-to-host VMs when upgrade of the host is finished or after upgrade of all hosts finished?

Reply:
Of all host is finished, because we can't start any VM, because of `cluster_maintanence` cluster scheduling policy.

Reply:
Hmm, I remember Yaniv talked about this requirement, that we should restart such VMs as soon as possible (for example if OpenShift is running on top of oVirt and we need to shutdown one of master node VM during host upgrade we should start that VM again right after host is successfully upgraded). Or did I misunderstood?

@mykaul @mgoldboi

Reply:
This is indeed sub-optimal. The use case I know of is multiple OpenShift masters, each pinned to a specific host. So once the first host is upgraded, you want to re-run the master pinned to it, so you can safely proceed to the next host.

In essence, this is going to be the situation in every VM level HA, where you pin the HA VMs to specific hosts.

Reply:
I've talked about this issue with Martin Sivak and he told me that first they should use affinity rather then pinning, so we should add support for affinity at first place.

Then, the `cluster_maintainence` policy is not so important to be enabled, so we can workaround this issue, that we don't enable that policy in case user use pinned non-migratable VMs.